packages/back-end/src/controllers/experimentLaunchChecklist.ts

```diff
@@ -135,9 +134,8 @@ export async function putManualLaunchChecklist(
     throw new Error("Could not find experiment");
   }
 
-  const envs = experiment ? getAffectedEnvsForExperiment({ experiment }) : [];
-
-  if (!context.permissions.canRunExperiment(experiment, envs)) {
+  const updatedExperiment = { ...experiment, manualLaunchChecklist: checklist };
+  if (!context.permissions.canUpdateExperiment(experiment, updatedExperiment)) {
```

Comment:
The 2nd argument should just be the fields that are changing, not the entire updated experiment.